### DISCUSSION TITLE
How to configure a database for Kong Gateway

### DISCUSSION THREAD

**Discussion by vbudilov** [Kong/kong#8298] [Help] [answered]
I need to add two plugins to my Kong installation — api-keys and jwt. From what I’m seeing the former would benefit from a DB installation since I might have more than a thousand api keys (the  latter would use AWS Cognito so it’s not in the same boat). I’ve been trying to find docs that talk about adding the DB to an existing Kong installation but can’t find it. Can anyone point it out to me?

**Comment by dndx** [comment, marked as answer]
Hello @vbudilov,

Assuming you are using DB-less right now and would like to switch over to DB mode. You can follow the [Quickstart guide](https://docs.konghq.com/gateway/2.7.x/get-started/quickstart/) to configure Kong to use a database, then use the `kong config db_import` command to import your existing `.yml` config into the new database. No data loss should occur.
